packages/babel-generator/test/fixtures/sourcemaps/function-identifier-name/output.js

```diff
@@ -0,0 +1,24 @@
+function fn() {}
+
+export default function () {}
+var a = {
+  fn() {},
+
+  fn: function () {},
+  [fn]: function () {},
```

Comment:
We need to duplicate these for the computed method forms: `[fn]() { }` shouldn't have an identifier, and I'm worried it'll be `fn`.